style.css

```diff
@@ -10,6 +10,9 @@ body {
 
 #aboutText {
 	margin-top: 2rem;
+	font-size: 14px;
```

Comment:
Use `rem` instead of pixels, because `rem` also works on mobile just as it does on desktop.